operator/pkg/apis/istio/v1alpha1/values_types.proto

```diff
@@ -1311,15 +1311,25 @@ message TelemetryV2Config {
   // Controls whether pilot will configure telemetry v2.
   google.protobuf.BoolValue enabled = 1;
 
+  TelemetryV2MetadataExchangeConfig metadata_exchange = 4;
+
   TelemetryV2PrometheusConfig prometheus = 2;
 
   TelemetryV2StackDriverConfig stackdriver = 3;
 }
 
+message TelemetryV2MetadataExchangeConfig {
+  // Controls whether enabled v8 runtime for metadata exchange filter.
+  google.protobuf.BoolValue enableV8 = 2;
```

Comment:
nit: I would call this `v8` or `v8Enabled` or something else. If it has `enabled` it should be at the end for consistency though

Reply:
You can also make it an enum with V8 and Native as first options in case we need to grow it. But it's fine to me as-is since this is mostly an internal flag.

Reply:
I updated it to `wasmEnabled` and remove the actual runtime from the option name.